connectivity/ble_weight_scale_nau7802/src/ble_weight_scale_nau7802.c

```diff
@@ -0,0 +1,146 @@
+/**
+ ****************************************************************************************
+ *
+ * @file ble_weight_scale_nau7802.c
+ *
+ * @brief BLE weight scale project source code.
+ *
+ * Copyright (C) 2012-2019 Dialog Semiconductor.
+ * This computer program includes Confidential, Proprietary Information
+ * of Dialog Semiconductor. All Rights Reserved.
+ *
+ ****************************************************************************************
+ */
+
+/**
+ ****************************************************************************************
+ * @addtogroup APP
+ * @{
+ ****************************************************************************************
+ */
+#include "rwip_config.h"             // SW configuration
+
+
+/*
+ * INCLUDE FILES
+ ****************************************************************************************
+ */
+
+#include "app_api.h"
+#include "ble_weight_scale_nau7802.h"
+#include "app_wsss.h"
+#include "nau7802.h"
+#include "custs1_task.h"
+#include "user_custs1_def.h"
+#include "arch_console.h"
+
+/*
+ * VARIABLE DEFINITIONS
+ ****************************************************************************************
+ */
+uint8_t conidx																				__SECTION_ZERO("retention_mem_area0"); //@RETENTION MEMORY
+timer_hnd timer_id																		__SECTION_ZERO("retention_mem_area0"); //@RETENTION MEMORY
+struct wss_wt_meas* meas;
+
+/*
+ * FUNCTION DEFINITIONS
+ ****************************************************************************************
+*/
+
```

Comment:
Add API header @brief @param

Reply:
added #3096600 to PR (I added the API header in .h file)